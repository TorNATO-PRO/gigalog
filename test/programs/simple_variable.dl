parent(john, mary).
parent(mary, bob).

has_child(X) :- parent(X, Y).
