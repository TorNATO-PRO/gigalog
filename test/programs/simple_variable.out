has_child(john)
has_child(mary)
parent(john,mary)
parent(mary,bob)
